components/scream/scripts/jenkins/mappy_setup

```diff
@@ -0,0 +1,5 @@
+module load sems-env sems-python/3.5.2
+
+source ./scream/components/scream/scripts/jenkins/sandia_son_proxy
```

Comment:
Mappy is currently on the SRN unfortunately.

Reply:
the son proxy is just an alias to the srn one. So sonproxy or wwwproxy doesn't matter. The IT folks confirmed that to me. Your next question might then be "why do we have an alias at all?". Well, good question.

Reply:
@bartgol , is that a new change? Using the right proxy for your network definitely mattered in the past.

Reply:
Just tested it, you're right!

Reply:
Yes, I was confused too when they told me. I found out a couple of months ago.